app/controllers/macros_controller.rb

```diff
@@ -1,8 +1,12 @@
 class MacrosController < ApplicationController
-  before_action :authorize_deployer!
-  before_action :authorize_admin!, only: [:new, :create, :edit, :update, :destroy]
+  include CurrentProject
```

Comment:
All this situations have been addressed in the following PR: [#567](https://github.com/zendesk/samson/pull/567)